worker/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google-scraper/database"
+	_ "google-scraper/initializers"
+	"google-scraper/worker/jobs"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocraft/work"
+	"github.com/beego/beego/v2/server/web"
+)
+
+var crawlingJobName string
+var enqueuerName string
```

Comment:
Do we need this? I don't see you use this anywhere outside this file.

Reply:
In the `init()` I am assigning value to  `enqueuerName`, `crawlingJobName` variable so . Then access these variable from the `main` function. Thats why I need to define it.